.github/actions/prepare-uberjar-artifact/action.yml

```diff
@@ -9,7 +9,7 @@ runs:
       run: sha256sum ./target/uberjar/metabase.jar > SHA256.sum
       shell: bash
     - name: Upload JARs as artifact
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v2.3.1
```

Comment:
I'm not sure why this is necessary as `@v2` already implies the latest `@v2.x.x`, which is `2.3.1` according to https://github.com/actions/upload-artifact/tags.

Reply:
Good catch. Didn't knew that. Thanks

So, I'll be closing this PR